Source/ChocolateyGui.Shared/ChocolateyGui.Shared.csproj

```diff
@@ -31,9 +31,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="chocolatey, Version=0.10.7.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb, processorArchitecture=MSIL">
-      <HintPath>..\packages\chocolatey.lib.0.10.7\lib\chocolatey.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="chocolatey, Version=0.10.8.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb, processorArchitecture=MSIL">
+      <HintPath>..\packages\chocolatey.lib.0.10.8\lib\chocolatey.dll</HintPath>

```

Comment:
not really a change request, but something I notice:
I don't like having path hierachies in nuget includes - maybe we should replace `..\packages` with something like `$(NuGetReferenceDir)` and then set the Property in a `Common.props` file?
- I'm working with 'global' nuget repository quite a bit - and therefore the `..\packages\..` paths just aren't correct (in my case)

Reply:
This is all managed with NuGet/Visual Studio, and these are hint paths. Anyway, when we upgrade it to Visual Studio 2017 project files, it will become much, much simpler.